src/Mod/Sketcher/Gui/DrawSketchHandler.cpp

```diff
@@ -406,12 +406,7 @@ DrawSketchHandler::suggestedConstraintsPixmaps(std::vector<AutoConstraint>& sugg
                 break;
         }
         if (!iconType.isEmpty()) {
-            qreal pixelRatio = 1;
-            Gui::View3DInventorViewer* viewer = getViewer();
-            if (viewer) {
-                pixelRatio = viewer->devicePixelRatio();
-            }
-            int iconWidth = 16 * pixelRatio;
+            constexpr int iconWidth = 16;

```

Comment:
Why? With 200% scaling the suggested icons are ridiculously small

Reply:
Yes I understood that later. However it does not work in this case. The suggested constraints are not scaled. So on my win11 200% scaling laptop the icons are very small.

Reply:
Should be fixed then somehow, we can restore that to fix the regression now, but we really must avoid having DPI awareness on higher levels of abstraction.

Reply:
@captain0xff the second regression (the one that is not fixed by your PR) can be fixed just by reverting this change. Perhaps that will give you hint as to why that happens.